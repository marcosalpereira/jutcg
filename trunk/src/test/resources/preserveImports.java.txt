package br.gov.qqpacote;

import br.gov.some.Class;

public class Tested {
    
}