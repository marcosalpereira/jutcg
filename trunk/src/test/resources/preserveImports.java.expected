package br.gov.qqpacote;

import br.gov.some.Class;
import static org.easymock.EasyMock.*;
import org.junit.*;
import java.util.*;
import static org.junit.Assert.*;

/**
 * Testes unitarios para a classe {@link Tested}.
 */
public class TestTested  {

/**
 * Classe sendo testada {@link Tested}.
 */
private Tested tested;
}